Classes/Backend/Form/Element/InputText.php

```diff
@@ -48,7 +48,6 @@ public function render() {
 	public function renderHtml($name, $value, $options) {
 		$width = $options['width'];
 		// TODO: das Feld beachten!
-	 	$maxlength = $options['maxlength'];
```

Comment:
I'd prefer to have a ticket for that instead of unused/not-yet-used code lines. Should I open a GitHub ticket for that?